@xen-orchestra/web/src/pages/pools.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="pools">
+    <UiCard class="pools-table">
```

Comment:
Be careful, here the class is the same as in the component, the styles conflict and make the link in the table span two lines:
![image](https://github.com/user-attachments/assets/89c79cc8-5b26-4708-bf79-dd73a986c983)

Suggestion:
```suggestion
    <UiCard class="pools-table-card">
```

After:
![image](https://github.com/user-attachments/assets/7a5a51be-0597-4696-9338-431c6fc0f5a8)